yb-voyager/src/tgtdb/target.go

```diff
@@ -95,3 +98,10 @@ func generateSSLQueryStringIfNotExists(t *Target) string {
 	}
 	return SSLQueryString
 }
+
+func GetRedactedTarget(t *Target) *Target {
+	redactedTarget := *t
+	redactedTarget.Uri = utils.GetRedactedURLs([]string{t.Uri})[0]
+	redact.Redact([]string{"Password"}, &redactedTarget)
```

Comment:
How about setting the Password field directly ;-) 

    redactedTarget.Password = "XXX"

Then you can remove the dependency on the `redact` module.